s6/debian-root/etc/services.d/pihole-FTL/run

```diff
@@ -9,13 +9,13 @@ rm /run/pihole/FTL.sock 2> /dev/null
 mkdir -pm 0755 /run/pihole
 [[ ! -f /run/pihole-FTL.pid ]] && install /dev/null /run/pihole-FTL.pid
 [[ ! -f /run/pihole-FTL.port ]] && install /dev/null /run/pihole-FTL.port
-[[ ! -f /var/log/pihole-FTL.log ]] && install /dev/null /var/log/pihole-FTL.log
-[[ ! -f /var/log/pihole.log ]] && install /dev/null /var/log/pihole.log
+[[ ! -f /var/log/pihole/pihole-FTL.log ]] && install /dev/null /var/log/pihole/pihole-FTL.log
```

Comment:
This log is now named just `FTL.log` (`/var/log/pihole/FTL.log`).

Reply:
Not yet, it isn't. It has not been merged https://github.com/pi-hole/FTL/pull/1375

Reply:
You're right... It will change after merge.

Reply:
Now it's merged

Reply:
```suggestion
[[ ! -f /var/log/pihole/FTL.log ]] && install /dev/null /var/log/pihole/FTL.log
```